SingleSampleQc.wdl

```diff
@@ -2,10 +2,10 @@ version 1.0
 
 ## Copyright Broad Institute, 2018
```

Comment:
Maybe this should say `Portions Copyright Broad Institute` at this point!

Reply:
fixed